tests/test_CustomRender.py

```diff
@@ -0,0 +1,63 @@
+# """Test TiTiler Custom Render Params."""
+
+from dataclasses import dataclass
+from typing import Optional, Union
+
+import numpy
+
+from titiler.dependencies import RenderParams
+from titiler.endpoints import factory
+
+from .conftest import DATA_DIR, parse_img
+
+from fastapi import FastAPI, Query
+
+from starlette.testclient import TestClient
+
+
+@dataclass
+class CustomRenderParams(RenderParams):
+
+    output_nodata: Optional[Union[str, int, float]] = Query(
+        None, title="Tiff Ouptut Nodata value",
+    )
+    output_compression: Optional[str] = Query(
+        None, title="Tiff compression schema",
+    )
+
+    def __post_init__(self):
+        super().__post_init__()
+        if self.output_nodata is not None:
+            self.kwargs["nodata"] = (
+                numpy.nan if self.output_nodata == "nan" else float(self.output_nodata)
+            )
+        if self.output_compression is not None:
+            self.kwargs["compress"] = self.output_compression
+
```

Comment:
cc @neda-dtu 
this is an example of custom RenderParams that add `output_nodata` and `output_compression` to the available query options (you could also hardcode some options).